keps/sig-release/2572-release-cadence/README.md

```diff
@@ -0,0 +1,1022 @@
+>         releases in April, August, December?
+>         releases in March, July, November?
+>         releases in February, June, October?
+>         releases in January, May, September?
+>     asymmetry with some explicit downtimes using the ~3 months we'd gain for things like triage/roadmap planning, personal downtime, stability efforts? Eg: late November, December, early January and also August have typically been slower times already.
+>         quiet December, dev activating more January through April release
+>         dev May through July release
+>         August stability scrubbing
+>         dev September through early December release
+>     some other explicit stability effort periods like taken in August 2020, formalized into the release cadence as a longer code freeze?
+>
+> Depending on how we lay out the annual calendar, we might also get some benefit (or complexity) relative to the golang release cycle and our need to update golang twice on average across the patch release lifecycle of each release. This may also compound relative to distributors release cadences, support lifecycles, their stabilization and lead time between content selection and release, and their balancing interoperability across a larger set of dependencies where those are tied to specific months on the annual calendar.
+
+@neolit123:
+
+> Golang has a "symmetric" model, so i think k8s should do the same.
+> the "symmetric" choice however, would require more discipline and availability from contributors, so my vote here is to try "symmetric" and if it fails (maybe after one year) go "asymmetric".
+
+@mpbarrett:
+
+> Which months (ie April, August, Dec would be every 4 months from the beginning of a year) would be important for me as a user to know.
+
+@OmerKahani:
+
+> @tpepper for your question - the best month for us will be March, July, November-December.
+
+@khenidak:
+
+> @tpepper would be great if we add to this post typical kubecon(s) schedule, since most the community (those who are reviewing, approving, building, releasing changes) is also heavily engaged in these events.
+
+@jberkus:
+
+> My vote is for symmetric releases in April, August, and December. While it's tempting to make December an "off month", development does happen all the time, and if it's not on a release, what is it on?
+>
+> That would be a reason for my 2nd choice, which would be Symmetric March, July, November, which puts Slow December at the beginning of the cycle instead of the end. However, that's mainly a benefit for working around Kubecon November, and there's no good reason to believe that Kubecon will be happening in November 2 years from now; it might be September or October instead.
+
+#### TODO Impact to existing deadlines
+
+@cblecker:
+
+> We should also talk about how this may impact things like code freeze (longer feature freeze with only bug/scale/stability fixes?).
```

Comment:
We continuously improve our release schedule together with the currently available processes. I think we should not go too deep into the rabbit whole here, and explicitly mention that it's in the responsibility of the SIG to decide the deadlines within a cycle on a per-release basis.